content/general/changelog.md

```diff
@@ -6,6 +6,10 @@ weight: 4
 
 Below you can find a full list of changes to VU, corresponding to every VU update, with the newest update at the top. Updates that have the `ONLY IN DEV` tag next to them are currently only available in the `dev` branch.
 
+{{% changeset 19412 "11/09/2022" true %}}
+- VeniceEXT was updated to `1.10.0` ([changelog](/vext/changelog/#1_10_0)).
+- Implementation of the ModSettings UI.
```

Comment:
This is a bit tech-y and specific. Should be something like `Add initial support for adjusting mod settings from the VU settings menu.`

Reply:
done